docs/YouboraSessions.md

```diff
@@ -0,0 +1,9 @@
+# Youbora sessions integration (javascript)
+
+## To integrate youbora sessions follow this documentation:
+https://documentation.npaw.com/npaw-integration/docs/application-analytics-custom-events-data-collection
+This means: create a plugin instance that can be accessed globally, and call fireEvent() and fireNav() to report what you want.
```

Comment:
need to add part on how to pass plugin instance to player on the plugin config

Reply:
Not really, since sessions plugin will be instantiated before the plugin inside the player, the first one will write the data we need on local/session storage and the second one will collect it from there, saving some code to integrate it